site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc

```diff
@@ -55,7 +69,6 @@ function(user, registration, jwt) {
 }
 ```
 
-Make sure you pick an email address for a domain you control to avoid malicious attacks. This will run your lambda twice.
-
-This capability is available beginning in version `1.31.0`.
+Make sure you pick an email address for a domain you control to avoid malicious attacks. Modifying the username or email address will cause your lambda to be run two times.
```

Comment:
Not exactly. It will only be run again if you modify the linking strategy data and an existing user is found that matches that data. For example, if you modify the email and we do not find an existing user that matches the new email, then the reconcile will not be called again because there is really no point. The second call is only to provide the opportunity for the lambda to act on a newly matched user that would not have been there during the first pass through the reconcile lambda.

Reply:
Okay, tweaked it to include your explanation.